GViewCore/src/View/GridViewer/Instance.cpp

```diff
@@ -0,0 +1,96 @@
+#include "GridViewer.hpp"
+
+using namespace GView::View::GridViewer;
+using namespace AppCUI::Input;
+
+Config Instance::config;
+
+Instance::Instance(const std::string_view& name, Reference<GView::Object> obj, Settings* settings) : settings(nullptr)
+{
+    this->obj  = obj;
+    this->name = name;
+
+    // settings
+    if ((settings) && (settings->data))
+    {
+        // move settings data pointer
+        this->settings.reset((SettingsData*) settings->data);
+        settings->data = nullptr;
+    }
+    else
+    {
+        // default setup
+        this->settings.reset(new SettingsData());
+    }
+
+    if ((this->settings) && (this->settings->content))
```

Comment:
Nu prea imi e clar ce e acle content ? 
Idea cu Settings era ca sa il poti seta direct din pluginul de tip --> nu sa trebuiasca sa dai ceva care trebuei reinterpretat. Problema e ca daca altcineva vrea sa foloseasca GridViewer-ul trebuei sa stie exact ce trebuie sa puna in acel content.

Reply:
fixed